tests/dns_test.toit

```diff
@@ -32,19 +33,43 @@ ipv6_address_test:
   expect_equals "102:304:506:708:90a:b0c:d0e:f10" addr.stringify
 
 ipv6_dns_test:
-  dns_query := DnsQuery_ "www.rwth-aachen.de"
-  ipv6 := dns_query.get --server="8.8.8.8" --accept_ipv6=true
-  expect ipv6.stringify == "2a00:8a60:450:0:0:0:107:63"
+  ipv6 := dns_lookup "www.rwth-aachen.de" --accept_ipv6=true --accept_ipv4=false
```

Comment:
Add comment explaining that the returned IP could change in which case the test should be adapted.